lib/rubocop/cop/sorbet/sigils/enforce_sigil_order.rb

```diff
@@ -28,6 +28,8 @@ module Sorbet
       # Only `typed`, `(en)?coding`, `warn_indent` and `frozen_string_literal` magic comments are considered,
       # other comments or magic comments are left in the same place.
       class EnforceSigilOrder < ValidSigil
+        include RangeHelp
```

Comment:
Where is this coming from and what does it do?

Reply:
It gives us access to the `source_range` method https://github.com/rubocop-hq/rubocop/blob/v0.85.0/lib/rubocop/cop/mixin/range_help.rb#L11

We need a range for`corrector.remove`